lib/refile/backend/file_system.rb

```diff
@@ -34,8 +34,12 @@ def upload(uploadable)
         Refile.verify_uploadable(uploadable, @max_size)
 
         id = @hasher.hash(uploadable)
-        IO.copy_stream(uploadable, path(id))
-
+        begin
+          IO.copy_stream(uploadable, path(id))
+        rescue IOError
+          uploadable.open
+          retry
```

Comment:
Good point. We'll investigate further. When uploadable is a Tempfile, it always comes through as closed.